assets/prefix-vendor-static-eval.js

```diff
@@ -0,0 +1,3 @@
+if (typeof window !== 'undefined' && window && window.performance && window.performance.mark) {
```

Comment:
Why do we need to check `window` here at all? Seems like this would work just the same:

```suggestion
if (typeof performance !== 'undefined' && tyepof performance.mark === 'function') {
```

Reply:
Thanks! You are right, I don't need to check window for this case. I got used to check window first at node environment. I have updated the code.